apps/app/src/app/pages/(components)/components/(breadcrumb)/breadcrumb.preview.ts

```diff
@@ -148,15 +148,15 @@ export const defaultSkeleton = `
 <nav hlmBreadcrumb>
 	<ol hlmBreadcrumbList>
 		<li hlmBreadcrumbItem>
-			<a hlmBreadcrumbLink href="/home">Home</a>
+			<a hlmBreadcrumbLink link="/home">Home</a>
 		</li>
 		<li hlmBreadcrumbSeparator></li>
 		<li hlmBreadcrumbItem>
 			<hlm-breadcrumb-ellipsis />
 		</li>
 		<li hlmBreadcrumbSeparator></li>
 		<li hlmBreadcrumbItem>
-			<a hlmBreadcrumbLink hlmL href="/components">Components</a>
+			<a hlmBreadcrumbLink hlmL link="/components">Components</a>
```

Comment:
Not sure where this `hlmL` attribute originally came from, but I don't believe it serves any purpose, so we can remove it 😊

```suggestion
			<a hlmBreadcrumbLink link="/components">Components</a>
```

Reply:
Fixed that with the latest commit. Nice catch!